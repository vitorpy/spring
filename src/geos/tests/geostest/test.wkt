version https://git-lfs.github.com/spec/v1
oid sha256:b6b51655f67f4b48b9cbb78d77e8487ea62105ec005db4940fd93633a8f5495d
size 110542
