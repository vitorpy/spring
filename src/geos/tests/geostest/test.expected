version https://git-lfs.github.com/spec/v1
oid sha256:15583688ca83a24f43239d8139663376ba8c7f44a1f99a57f5a7c40bc6d6a233
size 365213
